src/App.js

```diff
@@ -37,8 +105,30 @@ class App extends React.Component {
           <h3>High Card 🚀</h3>
           {currCardElems}
           <br />
-          <button onClick={this.dealCards}>Deal</button>
+          <button
+            onClick={() => {
+              this.state.roundsLeft > 0 ? this.dealCards() : this.resetGame();
```

Comment:
Since 0 is falsy, and other numbers are truthy, you can do 
```suggestion
              this.state.roundsLeft ? this.dealCards() : this.resetGame();
```